src/DotNet.RateLimiter/ServiceCollectionExtension.cs

```diff
@@ -49,11 +49,7 @@ public static void AddRateLimitService(this IServiceCollection services, IConfig
             else
             {
                 services.AddMemoryCache();
-                services.AddSingleton(new AsyncKeyedLocker<string>(o =>
-                {
-                    o.PoolSize = 20;
```

Comment:
Is the ``PoolSize`` handled internally in the new AsyncLock version?

Reply:
Yes it is. You can either handle it yourself or rely on the defaults. Currently I feel this combination is the best, but perhaps I will one day find a more optimal combination and change the defaults. We can revert this part if you want to retain control.

Reply:
@MarkCiliaVincenti I think better to keep visibility here, could you please revert this part?

Reply:
Done